proto/process/agent.proto

```diff
@@ -1760,3 +1761,11 @@ message ContainerRecommendations {
 	ResourceList upperBound = 4;
 	ResourceList uncappedTarget = 5;
 }
+
+message ContainerConditions {
```

Comment:
We'll use an array of those and they represent the VPA state so should this be renamed `VerticalPodAutoscalerCondition` instead?

Reply:
[dd434ab](https://github.com/DataDog/agent-payload/pull/227/commits/dd434abd07f7fbd3fdd02458c4e8e415276254c8)